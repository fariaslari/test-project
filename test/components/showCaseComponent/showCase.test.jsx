import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ShowCase from './../../../src/components/showCaseComponent/ShowCase';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('ShowCase', () => {
  it('should render correctly', () => {

    const output = shallow(
        <ShowCase />
    );
    
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});