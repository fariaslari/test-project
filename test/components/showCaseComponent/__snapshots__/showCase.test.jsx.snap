// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowCase should render correctly 1`] = `
<div
  className="showCase"
/>
`;
